<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="3" tests="12" name="USER_Admin_Reports.Marketing_campign" time="124.283" errors="0" timestamp="2024-12-11T18:24:34 IST" skipped="0">
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Username" time="0.755"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Password" time="0.805"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Captcha" time="0.777"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="SUBMIT_Button" time="0.303"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Remove_PouP" time="0.753"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Click_On_Reports" time="4.658"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Click_On_Marketing_Campign" time="0.126"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Check_Data__Of_Dealer_parent_Group_Wise" time="51.451">
    <failure message="&#039;Dealer parent Group Wise&#039; does not contain any data." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 'Dealer parent Group Wise' does not contain any data.
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.Marketing_campign.checkDataPresence(Marketing_campign.java:365)
at USER_Admin_Reports.Marketing_campign.Check_Data__Of_Dealer_parent_Group_Wise(Marketing_campign.java:90)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Check_Data__Of_Dealer_parent_Group_Wise -->
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Check_Data__Of_Zone_And_Region_Wise" time="7.511"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Check_Data__Of_Workshop_Wise" time="49.486">
    <failure message="&#039;Workshop Wise&#039; does not contain any data." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 'Workshop Wise' does not contain any data.
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.Marketing_campign.checkDataPresence(Marketing_campign.java:365)
at USER_Admin_Reports.Marketing_campign.Check_Data__Of_Workshop_Wise(Marketing_campign.java:210)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Check_Data__Of_Workshop_Wise -->
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Check_Data__Of_Day_Wise" time="7.657"/>
  <system-out/>
  <testcase classname="USER_Admin_Reports.Marketing_campign" name="Comparison_Of_All_Tab_Text" time="0.001">
    <failure message="Assertion failed: Zone Region Texts do not match Workshop Wise Texts!" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion failed: Zone Region Texts do not match Workshop Wise Texts!
at org.testng.Assert.fail(Assert.java:99)
at USER_Admin_Reports.Marketing_campign.Comparison_Of_All_Tab_Text(Marketing_campign.java:325)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Comparison_Of_All_Tab_Text -->
  <system-out/>
</testsuite> <!-- USER_Admin_Reports.Marketing_campign -->
